draft-ietf-httpbis-http2bis.xml

```diff
@@ -277,7 +277,9 @@
       <section anchor="versioning">
         <name>HTTP/2 Version Identification</name>
         <t>
-          The protocol defined in this document has two identifiers.
+          The protocol defined in this document has two identifiers. Creating a
+          connection based on either implies the use of the transport, security,
```

Comment:
I realize now that "security" does not hold for both.  We might need to explicitly say that "h2c" does not provide the security properties of "h2".